cloudflare/resource_cloudflare_api_token_test.go

```diff
@@ -0,0 +1,120 @@
+package cloudflare
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccAPIToken(t *testing.T) {
+	rnd := generateRandomResourceName()
+	resourceID := "cloudflare_api_token." + rnd
+	zoneID := os.Getenv("CLOUDFLARE_ZONE_ID")
+	permissionID := "82e64a83756745bbbb1c9c2701bf816b" // DNS read
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAPITokenConfig(rnd, rnd, permissionID, zoneID, true),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceID, "name", rnd),
+					resource.TestCheckResourceAttr(resourceID, "request_ip_in.0", "10.0.0.0/8"),
```

Comment:
Let's swap out these internal IP ranges for the reserved IP ranges mentioned in [rfc5737](https://tools.ietf.org/html/rfc5737)